.env.default

```diff
@@ -0,0 +1,6 @@
+FOURSQUARE_CLIENT_SECRET=
+FOURSQUARE_CLIENT_ID=
+FLASK_APP=fourstoryFlask
+SECRET_KEY=
```

Comment:
is the secret key getting set anywhere? or does it read automatically from the environment?

Reply:
This is just the .env.default file showing it's necessary for development; It's set locally in my .env file and passed in using GCP's secret manager in prod.